packages/vats/src/provisionPool.js

```diff
@@ -62,35 +82,69 @@ const makeBridgeProvisionTool = sendInitialPayment => {
             myAddressNameAdmin,
           ),
         ]);
-
+        provisionedCount += 1n;
+        publishMetrics();
         console.info('provisioned', address, powerFlags);
       },
     });
   return makeHandler;
 };
 
 /**
- * @typedef {StandardTerms & {
+ * @typedef {StandardTerms & GovernanceTerms<{
+ *    PerAccountInitialAmount: 'amount',
+ *   }> &{
  *   perAccountInitialAmount: Amount<'nat'>,
```

Comment:
```suggestion
 *    PerAccountInitialAmount: Amount<'nat'>,
```

are both perAccountInitialAmount and PerAccountInitialAmount expected here?

Reply:
only `PerAccountInitialAmount`